src/components/Tabs/Tab/Tab.tsx

```diff
@@ -51,8 +51,8 @@ export const Tab = React.forwardRef<HTMLLIElement, TabProps>((props, ref) => {
       aria-current={isActive}
     >
       <span role="link">{title}</span>
-      <div className={classes.focusIndicatorWrapper} tabIndex={-1}>
-        <div className={classes.focusIndicator} tabIndex={-1} />
+      <div className={classes.focusIndicatorWrapper} tabIndex={-1} aria-hidden={'true'}>
```

Comment:
The parent `aria-hidden` is enough, since it affects its children as well
So no need to have both

Reply:
I removed the descendant aria-hidden and I signed the last commit. Is it enough for signing?